pubsub/azurepubsub/azurepubsub.go

```diff
@@ -117,8 +118,10 @@ const Scheme = "azuresb"
 //   - The URL's host+path is used as the topic name.
```

Comment:
The URL above should be `azuresb://mytopic?subscription=`

Reply:
Ooh, good catch. Done.